src/js/components/RadioButton/doc.js

```diff
@@ -91,6 +91,11 @@ export const themeDoc = {
     type: 'string',
     defaultValue: '100%',
   },
+  'radioButton.check.size': {
+    description: 'The check size of the RadioButton.',
+    type: 'string',
```

Comment:
would this support t-shirt sizing of small, medium, large?

Reply:
I changed the type to `number` so isn't being supported, but if we change the type to  `number | string` this can be implemented. What do you think?

Reply:
I think that if we stick with eric's comment below it will naturally take care of it.